glean-core/rlb/tests/fog_simple.rs

```diff
@@ -0,0 +1,89 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! This test should be conceptually similar to how the RLB is used from e.g. FOG.
```

Comment:
Do we need to mention FOG at all here? It seems very out of context. We have a [full init example](https://searchfox.org/glean/source/glean-core/examples/sample.rs). Maybe we should remove the example and just turn that into a `simple_usage_test.rs`?

Reply:
integration tests run as part of `cargo test`, which is kinda nice.
(I see them similar to our sample apps in Kotlin and Swift).
FOG is mentioned because the patterns here are modeled the way we use them in FOG.